source/_posts/2022-12-20-year-of-voice.markdown

```diff
@@ -0,0 +1,65 @@
+date: 2022-12-20 00:00:00
+date_formatted: "December 20, 2022"
+author: Paulus Schoutsen
+author_twitter: balloob
+comments: true
+categories:
+- Announcements
+og_image: /images/blog/2022-12-20-year-of-voice/conversation.png
+---
+
+_**TL;DR**: It is our goal for 2023 to let users control Home Assistant in their own language. Mike Hansen, creator of Rhasspy, has joined Nabu Casa to lead this effort. We’re starting off by building a [collection of intent matching sentences in every language](https://github.com/home-assistant/intents)._
+
+Usually the month of December is meant to reflect back. However, we already did that last month when we hosted the [State of the Open Home 2022](https://www.youtube.com/watch?v=D936T1Ze8-4). We didn’t only reflect, we also announced our focus for next year: 2023 is going to be the year of voice.
+
+<blockquote>It is our goal for 2023 to let users control Home Assistant in their own language.</blockquote>
+
+It's a big and bold goal, but achievable given the right constraints. The amount of work laid out for us can be summarised as follows:
+
+<center><img src='/images/blog/2022-12-20-year-of-voice/voice-work.png' alt='Spoken languages times possible actions' class='no-shadow' /></center>
+<br>
+
+Our #1 priority is supporting different languages. There are enough projects out there trying to create an English voice assistant. But for us that just doesn't cut it. People need to be able to speak in their own language, as that is the most accessible and only acceptable language for a voice assistant for the smart home.
+<!--more-->
+
+To keep the amount of work ahead of us manageable, we're going to limit the number of possible actions and focus on the basics of interacting with your smart home. No web searches, making calls or voice games. And definitely no "by the way"s!
+
+We are going to start with a few actions and build up the language models around that. Home Assistant supports 62 different languages in its user interface. And it's our goal to support all these languages with voice. We think that we can achieve that by leveraging Home Assistant's strongest asset: our community.
+
+## Our history with voice assistants
+
+If you follow the news, it might sound like voice assistants have failed. Amazon is set to [lose $10 billion on Alexa this year](https://arstechnica.com/gadgets/2022/11/amazon-alexa-is-a-colossal-failure-on-pace-to-lose-10-billion-this-year/) and is planning layoffs. Google too, is reducing its [support for Google Assistant](https://arstechnica.com/gadgets/2022/10/report-google-doubles-down-on-pixel-hardware-cuts-google-assistant-support/) as it’s trying to cut costs. The truth is that voice as the next computing platform that drives billions of dollars of extra revenue has failed. Instead, users mainly use their voice assistants to manage shopping lists, set timers, play music and control their home. Voice has failed being a source of revenue, it has not failed its users.
```

Comment:
```suggestion
If you follow the news, it might sound like voice assistants have failed. Amazon is set to [lose $10 billion on Alexa this year](https://arstechnica.com/gadgets/2022/11/amazon-alexa-is-a-colossal-failure-on-pace-to-lose-10-billion-this-year/) and is planning layoffs. Google too, is reducing its [support for Google Assistant](https://arstechnica.com/gadgets/2022/10/report-google-doubles-down-on-pixel-hardware-cuts-google-assistant-support/) as it’s trying to cut costs. The truth is that voice, as the next computing platform that drives billions of dollars of extra revenue, has failed. Instead, users mainly use their voice assistants to manage shopping lists, set timers, play music, and control their homes. Voice has failed being a source of revenue, it has not failed its users.
```